documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,33 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting interval between precopies
+
+You can set the precopy interval.
```

Comment:
Change to "You can set the interval between snapshots during the precopy stage of warm migration."